R/drawing-context.R

```diff
@@ -70,13 +70,13 @@ set_context_gp <- function(drawing_context, gp = NULL) {
 
 # update the fontface of a drawing context
 set_context_fontface <- function(drawing_context, fontface = "plain", overwrite = FALSE) {
-  fontface_old <- drawing_context$gp$fontface
+  fontface_old <- drawing_context$gp$font
```

Comment:
I think it would be better to use `font_old` instead of `fontface_old` here, so there is no confusion about that the variable contains.